### DISCUSSION TITLE
Option to group gifted subs

### DISCUSSION THREAD

**Discussion by slamy31** [Chatterino/chatterino2#4486] [Ideas]
Could we have an option to group gifted subs to avoid large amounts of spam? Or maybe just displaying the overall amount message and not the individuals that come after it?

**Comment by Felanbird** [comment]
> Or maybe just displaying the overall amount message and not the individuals that come after it?

This can be achieved with the filter system, you can apply the following filter to tabs to not see the gift messages
`!(flags.sub_message && message.content contains "gifted a tier")`

before:
![image](https://user-images.githubusercontent.com/41973452/228329400-d0ed51cb-abf0-466c-a02c-be1552ed626e.png)


after:
![image](https://user-images.githubusercontent.com/41973452/228329210-58b7ff25-d812-4586-9716-f40807254096.png)

**Reply by slamy31** [reply]
Would this remove single gift subs?

**Reply by Felanbird** [reply]
Yes
